app/src/main/java/com/mathewsachin/fategrandautomata/util/ImageLoader.kt

```diff
@@ -31,13 +32,15 @@ class ImageLoader @Inject constructor(
         return null
     }
 
-    private fun createPattern(FilePath: String): IPattern {
+    private fun createPattern(gameServer: GameServerEnum, FileName: String): IPattern {
+        val filePath = "$gameServer/${FileName}"
```

Comment:
Is the `${FileName}` necessary or would `$FileName` suffice? You're not calling any method or reading properties here.